keps/sig-node/0009-node-heartbeat.md

```diff
@@ -243,9 +255,19 @@ The changes in components logic (Kubelet, NodeController) should be done behind
 a feature gate. We suggest making that enabled by default once the feature is
 implemented.
 
+Beta:
+- Proved scalability/performance gain. We expect significant decrease in total
+etcd size (at least 2x) and no drop in any other scalability SLIs.
+
+GA:
+- Enabled by default for a release with no complaints.
```

Comment:
Does that imply 1.16?

Reply:
Given we're going to Beta in 1.14, I think we can do 1.15 still (assuming everything will be fine)?

Reply:
To advance to GA in 1.15, the work to make this GA would have to be done shortly after shipping 1.14, by which time few people would have used it, and probably none in production scenarios. 

If we have high confidence, it can GA in 1.15, but it's not as if it will be "enabled by default for a release" by the time that decision needs to be made.